iris-machine-learning-tutorial/Makefile

```diff
@@ -0,0 +1,36 @@
+export AWS_ACCESS_KEY_ID ?= test
+export AWS_SECRET_ACCESS_KEY ?= test
+export AWS_DEFAULT_REGION = us-east-1
+
+usage:       ## Show this help
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+
+install:     ## Install dependencies
+	@which awslocal || pip install awscli-local
+
+start:
+	localstack start -d
+
+stop:
+	@echo
+	localstack stop
+
+ready:
+	@echo Waiting on the LocalStack container...
+	@localstack wait -t 30 && echo Localstack is ready to use! || (echo Gave up waiting on LocalStack, exiting. && exit 1)
+
+logs:
+	@localstack logs > logs.txt
+
+run:
+	awslocal ecr create-repository --repository-name iris-machine-learning-tutorial | jq -r '.repository.repositoryUri'
+	docker build -t localhost:4510/iris-machine-learning-tutorial .
+	docker push localhost:4510/iris-machine-learning-tutorial:latest
+	awslocal lambda create-function --function-name iris-prediction --package-type Image \
+		--code ImageUri=localhost:4510/repo1 --role arn:aws:iam::000000000:role/lambda-r1 \
```

Comment:
Does the IAM role have to be created beforehand? I haven't tried this, not sure if this would work or would give an error, because the role is not there.

Reply:
Nope